tests/README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Before starting
 
-In order to run ansible-freeipa tests you will need to have `ansible`, `pytest` and `jmespath` installed on your machine. We'll call this local machine `controller`. `jmespath` is needed for the `json_query` filter.
+In order to run ansible-freeipa tests you will need to install the dependencies listed in the file `tests/requirements.txt` in your local machine. We'll call this local machine `controller`.
```

Comment:
Shouldn't this be `requirements-tests.txt`? Is the file in the tests folder?

Reply:
Right! I've changed the file but didn't changed the docs! Thanks.